tests/Fixtures/Entity/Bar.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="bars")
+ */
+class Bar
```

Comment:
Let's remove this entity and just use the `Category` fixture for Foo's relationships. It already has it's factory as a service so it will just work for the test.